app/grandchallenge/algorithms/templates/algorithms/algorithmimage_detail.html

```diff
@@ -39,7 +39,7 @@ <h2>Algorithm Container Image</h2>
         </a><br>
     {% endif %}
 
-    {% if object.is_manifest_valid and not object.is_desired_version and "change_algorithm" in algorithm_perms %}
+    {% if object.is_manifest_valid and not object.algorithm.image_upload_in_progress and not object.is_desired_version and "change_algorithm" in algorithm_perms %}
         <div class="my-2">Activating an image will result in this image being used for future algorithm runs:</div>
         <div class="mb-2">{% crispy image_activate_form %}</div>
```

Comment:
Do we also need a check in the form itself?

Reply:
We already check this in the form's clean method.